WordPress-iOS-Shared/Core/WPTableViewSectionHeaderFooterView.m

```diff
@@ -0,0 +1,243 @@
+#import "WPTableViewSectionHeaderFooterView.h"
+#import "WPTableViewCell.h"
+#import "WPStyleGuide.h"
+#import "NSString+Util.h"
+
+
+
+@interface WPTableViewSectionHeaderFooterView ()
+@property (nonatomic, strong) UILabel *titleLabel;
+@end
+
+
+@implementation WPTableViewSectionHeaderFooterView
+
+/*
+You should invoke the designated initializer of your base class here: initWithReuseIdentifier.
+If you aren't make the reuse identifier available for this class you can invoke with nil.
+
+On the same page you should override that default initializer to use this initializer with a default style set.
```

Comment:
You should add this:
```
- (instancetype)initWithReuseIdentifier:(NSString *)reuseIdentifier
{
return [self initWithReuseIndentifier:reuseIdentifier style:WPTableViewSectionStyleHeader]
}
```
to avoid people calling the super initialiser and not getting the state variable properly set and the subviews set.